src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
-// write your code here
+const populations = document.querySelectorAll('.population');
+let totalValue = 0;
+
+for (let i = 0; i < populations.length; i++) {
+  const value = populations[i].textContent.replace(/,/g, '');
+
+  if (!isNaN(Number(value))) {
+    totalValue += Number(value);
+  }
+
+  const averageValue = totalValue / populations.length;
```

Comment:
The calculation of `averageValue` should be moved outside the loop. Calculating the average inside the loop will result in incorrect values being calculated and displayed multiple times.